server/src/main/java/org/apache/cassandra/sidecar/cdc/CdcConfigImpl.java

```diff
@@ -179,7 +179,7 @@ public MinuteBoundConfiguration watermarkWindow()
     {
         // this prop sets the maximum duration age accepted by CDC, any mutations with write timestamps older than
         // the watermark window will be dropped with log message "Exclude the update due to out of the allowed time window."
-        return new MinuteBoundConfiguration(getInt(ConfigKeys.WATERMARK_SECONDS.lowcaseName, DEFAULT_WATERMARK_WINDOW), TimeUnit.SECONDS);
+        return new MinuteBoundConfiguration(getInt(ConfigKeys.WATERMARK_SECONDS.lowcaseName, DEFAULT_WATERMARK_WINDOW) / 60, TimeUnit.MINUTES);
```

Comment:
Should we change the resolution to Seconds instead? What if someone configures 30 seconds for the watermark? That integer would be truncated to zero right?

Reply:
We can change that but should that be minute granularity or seconds granularity is the question. Also, note that it chages the API if we change it to SecondBound

Reply:
Then we should change the ConfigKey to WATERMARK_MINUTES.

Reply:
We can, but that might break any existing configs that are being used

Reply:
I think changing the MinuteBound to SecondBound is better compared to changing configkey as it will break at compile time.